Podcast/BookmarkViewController.swift

```diff
@@ -30,20 +30,23 @@ class BookmarkViewController: UIViewController, UITableViewDelegate, UITableView
         bookmarkTableView.backgroundColor = .clear
         bookmarkTableView.separatorStyle = .none
         bookmarkTableView.showsVerticalScrollIndicator = false
-        bookmarkTableView.register(CardTableViewCell.self, forCellReuseIdentifier: "CardTableViewCellIdentifier")
+        bookmarkTableView.register(BookmarkTableViewCell.self, forCellReuseIdentifier: "BookmarkTableViewCellIdentifier")
         view.addSubview(bookmarkTableView)
-        bookmarkTableView.rowHeight = CardTableViewCell.height
+        bookmarkTableView.rowHeight = BookmarkTableViewCell.height
         bookmarkTableView.reloadData()
         
-        cards = fetchCards()
+        episodes = fetchEpisodes()
```

Comment:
Makes sense. That's what I did in my other views but I didn't write this view so I left it this way. I'll change that